gsocscrape/items.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+# Define here the models for your scraped items
```

Comment:
this is junk added by the generator to guide someone writing their code.  It doesnt need to be in our repository